### PR TITLE
Fix some Doctrine XML configurations

### PR DESCRIPTION
**Pull request by atailouloute**

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
👍

File: http_cache/esi.rst [hunk 1]
```diff
@@ -234,7 +234,7 @@ that must be enabled in your configuration:
         <?xml version="1.0" encoding="UTF-8" ?>
         <container xmlns="http://symfony.com/schema/dic/services"
             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-            xmlns:doctrine="http://symfony.com/schema/dic/framework"
+            xmlns:doctrine="http://symfony.com/schema/dic/doctrine"
```
Comment (xabbuh) [inline http_cache/esi.rst]: This change looks wrong to me. Instead, the namespace should be changed to `framework`:

```
xmlns:framework="http://symfony.com/schema/dic/framework"
```

**Review by xabbuh — CHANGES_REQUESTED** [review]

File: http_cache/esi.rst (hunk 1 above)
Comment (atailouloute) [inline http_cache/esi.rst]: @xabbuh Yes you are right (y), It's fixed

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you for the many good catches and improvements @atailouloute! To ease the merging of all the small fixes I will combine your PRs in a single PR (see #7737) and merge that one instead.

**Pull request closed without merging**